docs-ref-conceptual/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db.md

```diff
@@ -294,6 +294,8 @@ For more information about using Azure Cosmos DB and Java, see the following art
 
 * [Azure Cosmos DB: Create a document database using Java and the Azure portal][Build a SQL API app with Java]
```

Comment:
Could you please also provide the correct link to the item above, in this PR? 

https://docs.microsoft.com/en-us/azure/cosmos-db/create-sql-api-java